Mathlib/NumberTheory/ADEInequality.lean

```diff
@@ -194,7 +194,9 @@ theorem lt_three {p q r : ℕ+} (hpq : p ≤ q) (hqr : q ≤ r) (H : 1 < sumInv
 #align ADE_inequality.lt_three ADEInequality.lt_three
 
 theorem lt_four {q r : ℕ+} (hqr : q ≤ r) (H : 1 < sumInv {2, q, r}) : q < 4 := by
-  have h4 : (0 : ℚ) < 4 := by norm_num
+  have h4 : (0 : ℚ) < 4 :=
+  by
```

Comment:
note that this is not yet covered as the correct fix requires some more work (reflowing lines in a way that keeps them under 100 chars is slightly tricky)